PROTOCOL.md

```diff
@@ -564,22 +565,54 @@ The following is an example `rowIdHighWaterMark` action:
 }
 ```
 
+### Domain Metadata
+The domain metadata action contains a configuration (string-string map) for a named metadata domain. Two overlapping transactions conflict if they both contain a domain metadata action for the same metadata domain.
+
+There are two types of metadata domains:
+1. **User-controlled metadata domains** have names that start with anything other than the `delta.` prefix. Any Delta client implementation or user application can modify these metadata domains, and can allow users to modify them arbitrarily. Delta clients and user applications are encouraged to use a naming convention designed to avoid conflicts with other clients' or users' metadata domains (e.g. `com.databricks.*` or `org.apache.*`).
```

Comment:
How should a client handle user-controlled metadata domains, especially ones it doesn't recognize? 
I guess it should propagate the metadata domain unmodified on the write path, and ignore it on the read path, but the spec should probably say so?

Reply:
Looks good so far -- we can use new comments in those new sections to nitpick any wording changes there.